editor/src/clj/editor/defold_project.clj

```diff
@@ -793,35 +1198,33 @@
           (g/with-auto-evaluation-context evaluation-context
             (into {}
                   (map (fn [[resource old-node-id]]
-                         [resource
-                          (du/measuring resource-metrics (resource/proj-path resource) :collect-overridden-properties
-                            (g/collect-overridden-properties old-node-id evaluation-context))]))
+                         (pair resource
+                               (du/measuring resource-metrics (resource/proj-path resource) :collect-overridden-properties
+                                 (g/collect-overridden-properties old-node-id evaluation-context)))))
                   (:transfer-overrides plan))))
 
-        old-nodes-by-path (g/node-value project :nodes-by-resource-path)
-        rn-dependencies-evaluation-context (g/make-evaluation-context)
-        old-resource-node-dependencies (fn/memoize
-                                         (fn old-resource-node-dependencies [node-id]
-                                           (let [resource (g/node-value node-id :resource rn-dependencies-evaluation-context)]
-                                             (du/measuring resource-metrics (resource/proj-path resource) :find-old-reload-dependencies
-                                               (when-some [dependencies-fn (:dependencies-fn (resource/resource-type resource))]
-                                                 (let [save-value (g/node-value node-id :save-value rn-dependencies-evaluation-context)]
-                                                   (when-not (g/error? save-value)
-                                                     (dependencies-fn save-value))))))))
-        resource->old-node (comp old-nodes-by-path resource/proj-path)
+        old-evaluation-context (g/make-evaluation-context)
+        old-basis (:basis old-evaluation-context)
+        old-node-ids-by-proj-path (g/valid-node-value project :nodes-by-resource-path old-evaluation-context)
```

Comment:
Renamed some stuff here for consistency with the new `read-nodes` implementation and across the file. The `old-resource-node-dependencies` code was moved down as well.